CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ to [sourceforge feature requests](https://sourceforge.net/p/jabref/features/) by
 - [#1023](https://github.com/JabRef/jabref/issues/492) ArXiv fetcher now also fetches based on eprint id
 
 ### Fixed
+- Fixed "Open-folder" action in Windows not opening tge correct folder when folder path contained whitespaces
```

Comment:
The Changelog entry is NOT needed. JabRef worked fine in 3.2. Please remove the line.